static/js/dark_theme.ts

```diff
@@ -1,13 +1,28 @@
 import $ from "jquery";
 
 export function enable(): void {
+    if (typeof Storage !== "undefined") {
+        localStorage.setItem("theme-value", "dark-theme");
+    } else {
+        // Sorry! No Web Storage support..
+    }
```

Comment:
We don't need to use local storage for this; just include the user's theme choice in the `page_params` we send when rendering the stats page -- that'll always be correct without any additional data synchronization bugs to consider.